rosdep/python.yaml

```diff
@@ -4909,6 +4916,13 @@ python-sqlalchemy:
 python-sqlite:
   debian: [python-sqlite]
   ubuntu: [python-sqlite]
+python-statistics-pip:
```

Comment:
same

Reply:
This is basically a python2 port of the default python 3 statistics module. Unless you disagree, I don't really see any reason to make it available for python3?